app/src/main/java/com/mymeetings/android/view/widgets/CalendarEventWidgetService.kt

```diff
@@ -73,120 +70,72 @@ class CalendarEventWidgetRemoteViewFactory(
 
     override fun getViewAt(position: Int): RemoteViews {
         if (position != AdapterView.INVALID_POSITION && calendarEventsWithViewAlertType?.size ?: 0 > position) {
-            val calendarEventWithAlertPair = calendarEventsWithViewAlertType?.get(position)
             return RemoteViews(context.packageName, R.layout.item_meeting).apply {
-                calendarEventWithAlertPair?.let {
-                    val startTimeToEndTimeString = "${it.startTime} - ${it.endTime}"
-                    if (it.viewAlertType == ViewAlertType.RUNNING) {
-                        setTextViewText(R.id.runningTitleText, it.calendarEvent.title)
-                        setTextViewText(R.id.runningTimeText, startTimeToEndTimeString)
-                        setViewVisibility(R.id.upcomingLayout, View.GONE)
-                        setViewVisibility(R.id.runningLayout, View.VISIBLE)
-                    } else {
-                        setTextViewText(R.id.titleText, it.calendarEvent.title)
-                        setTextViewText(R.id.timeText, startTimeToEndTimeString)
-                        setTextViewText(R.id.timeLeftToStartText, it.timeLeftToStart)
```

Comment:
This way of accessing String fields like `timeLeftToStart ` in the UI was much readable and clearer than the current approach.